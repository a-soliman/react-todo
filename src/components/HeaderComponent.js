import React from 'react';
import { NavLink } from 'react-router-dom';

import NavComponent from './NavComponent';

const HeaderComponent = () => (
    <div>
        <div className='container'>
            <div className='row'>
                <div className='col text-center'>Todo</div>
            </div>
            <div className='row'>
                <NavComponent />
            </div>
        </div>
    </div>
);

export default HeaderComponent;