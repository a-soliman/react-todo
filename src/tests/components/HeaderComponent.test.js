import React from 'react';
import { shallow } from 'enzyme';
import HeaderComponent from '../../components/HeaderComponent';

describe('HeaderComponent', () => {
    test('Should render HeaderComponent correctly', () => {
        const wrapper = shallow(<HeaderComponent />);
        expect(wrapper).toMatchSnapshot();
    });
});