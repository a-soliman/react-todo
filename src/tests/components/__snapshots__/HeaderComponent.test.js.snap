// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderComponent Should render HeaderComponent correctly 1`] = `
<div>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col text-center"
      >
        Todo
      </div>
    </div>
    <div
      className="row"
    >
      <NavComponent />
    </div>
  </div>
</div>
`;
